receiver/vcenterreceiver/metadata.yaml

```diff
@@ -0,0 +1,374 @@
+    unit: "{MHz}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.resource_pool.cpu.shares:
+    enabled: true
+    description: The amount of shares of CPU in the resource pool.
+    unit: "{shares}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.vm.memory.ballooned:
+    enabled: true
+    description: The amount of memory that is ballooned due to virtualization.
+    unit: By
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.vm.memory.usage:
+    enabled: true
+    description: The amount of memory that is used by the virtual machine.
+    unit: MBy
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.vm.cpu.utilization:
+    enabled: true
+    description: The CPU utilization of the virtual machine.
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.vm.disk.usage:
+    enabled: true
+    description: The amount of storage space used by the virtual machine.
+    unit: By
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [disk_state]
+  vcenter.vm.disk.utilization:
+    enabled: true
+    description: The utilization of storage on the virtual machine.
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.vm.disk.latency.avg:
+    enabled: true
+    description: The latency of operations to the virtual machine's disk.
+    unit: µs
+    gauge:
+      value_type: int
+    attributes: [latency_direction]
+    extended_documentation: This latency is the sum of the device and kernel write latencies. Requires Performance Counter level 2 for metric to populate.
+  vcenter.vm.disk.latency.max:
+    enabled: true
+    description: The highest reported total latency (device and kernel times) over an interval of 20 seconds.
+    unit: ms
+    gauge:
+      value_type: int
+    attributes: []
+  vcenter.vm.disk.throughput:
+    enabled: true
+    description: The throughput of the virtual machine's disk.
+    unit: By/sec
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+
+  vcenter.vm.network.throughput:
+    enabled: true
+    description: The amount of data that was received or sent over the network of the virtual machine.
+    unit: By/sec
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [throughput_direction]
+  vcenter.vm.network.packets:
+    enabled: true
+    description: The amount of packets that was received or transmitted over the instance's network.
+    unit: "{packets/sec}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [throughput_direction]
+  vcenter.vm.network.usage:
```

Comment:
Are there counter versions of these that can be used instead?

Reply:
Same thing here https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/9224#discussion_r855234751